lib/redis/namespace.rb

```diff
@@ -239,6 +240,12 @@ class Namespace
     # Support 1.8.7 by providing a namespaced reference to Enumerable::Enumerator
     Enumerator = Enumerable::Enumerator unless defined?(::Enumerator)
 
+    @sadd_returns_boolean = true
+
+    class << self
+      attr_accessor :sadd_returns_boolean
+    end
+
```

Comment:
This isn't needed, right? Can we remove it?

Reply:
It's needed, as it provides the way to set and read the `sadd_returns_boolean` value. This is used by the `Redis` gem to determine whether or not to display that deprecation message.